update.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
Who runs this script? It might make sense to run this as part of `synth.py` rather than invoke synthtool from here. That would remove a manual step.

Reply:
As discussed offline, the proto will be moved to `third_party` after all, so everything should be automatic. 

Can we first merge it for now, and move it to `third_party` in a followup PR?

Reply:
@BenWhitehead 

FYI, we will move the proto to `third_party` of googleapis after all, looks like the benefits are really attractive..

Reply:
Syncing of `thrid_party` to https://github.com/googleapis/googleapis is usually the following night, and autosynth should run later that night.

So I guess it's up to you on the urgency.

Reply:
Let me do it in a followup PR.